### DISCUSSION TITLE
Will there be mac os support?

### DISCUSSION THREAD

**Discussion by EnderRobber101** [PassivePicasso/ThunderKit#100] [Q&A]
Will there be mac os support anytime soon?

**Comment by PassivePicasso** [comment]
It's really up to the Mac user community when it happens.  I don't have a Mac, nor any desire to own a mac, so I don't have a way to develop and test support.

Linux Support only required a few minor tweaks to the Import system and otherwise just worked, a community member did the leg work on that to get support enabled.  Like Mac OSX, I don't use Linux, so I couldn't really build that support without assistance.

**Comment by PassivePicasso** [comment]
The changes seem to mostly be around usages of Application.platform, and are largely about differences in the way the file system is setup.
https://github.com/PassivePicasso/ThunderKit/blob/master/Editor/Core/Data/ThunderKitSettings.cs#L284
https://github.com/PassivePicasso/ThunderKit/blob/master/Editor/Core/Config/ImportConfiguration.cs#L317

**Comment by PassivePicasso** [comment]
GetBitness also checks platform, but my understanding is that you can only use 64bit on Mac OSX, and therefor its not required, but it can be looked at here:
https://github.com/PassivePicasso/ThunderKit/blob/master/Editor/Core/Config/Common/GetBitness.cs
